source/pages/tutoriel/index.tsx

```diff
@@ -0,0 +1,223 @@
+						<details>
+							<summary className=" font-bold text-primary">
+								C’est quoi mon empreinte carbone ?
+							</summary>
+							<div className="ml-3.5 my-2">
+								<p>
+									La planète se réchauffe dangereusement, au fur et à mesure des
+									gaz à effet de serre que l'on émet.
+								</p>
+								<p>
+									Pas pour votre boulot ou vos études. Une seule exception :
+									votre trajet domicile-travail doit être inclus dans les km
+									parcourus.
+								</p>
+							</div>
+						</details>
+					</li>
+					<li className="my-1" id={'mesure'}>
+						<details>
+							<summary className=" font-bold text-primary">
+								Comment on la mesure ?
+							</summary>
+							<div className="ml-3.5 my-2">
+								<Trans i18nKey={'publicodes.Tutorial.slide2.p1'}>
+									Avec une unité au nom barbare : l'équivalent CO₂. Le dioxyde
+									de carbone , vous le connaissez : on l'expire toute la
+									journée, mais sans influence sur le climat.
+								</Trans>
+								<Trans i18nKey={'publicodes.Tutorial.slide2.p2'}>
+									Ce sont les machines qui font notre confort moderne qui en
+									rejettent massivement, à tel point qu'on le compte en milliers
+									de kilos par an et par personne, donc en{' '}
+									<strong>tonnes</strong> de CO₂e !
+								</Trans>
+							</div>
+						</details>
+					</li>
+					<li className="my-1" id={'objectif'}>
+						<details>
+							<summary className=" font-bold text-primary">
+								Quel est l’objectif à atteindre ?
+							</summary>
+							<div className="ml-3.5 my-2">
+								<Trans i18nKey={'publicodes.Tutorial.slide5.p1'}>
+									<p>
+										Nous devons diminuer notre empreinte climat au plus vite.
+									</p>
+									<p>
+										En France, ça consiste à passer de ~10 tonnes à{' '}
+										<strong>moins de 2 tonnes</strong> par an.
+									</p>
+								</Trans>
+								<Trans>Pour en savoir plus, tout est expliqué dans </Trans>
+								<a href="https://datagir.ademe.fr/blog/budget-empreinte-carbone-c-est-quoi/">
+									<Trans>cet article</Trans>
+								</a>{' '}
+								<Trans>(15 min de lecture)</Trans>
+							</div>
+						</details>
+					</li>
+					<li className="my-1" id={'categories'}>
+						<details id={'categories'}>
+							<summary className=" font-bold text-primary">
+								D’où vient mon empreinte ?
+							</summary>
+							<div className="ml-3.5 my-2">
+								<Trans i18nKey={'publicodes.Tutorial.slide6'}>
+									<p>
+										Prendre la voiture, manger un steak, chauffer sa maison, se
+										faire soigner, acheter une TV...
+									</p>
+									<p>
+										L'empreinte de notre consommation individuelle, c'est la
+										somme de toutes ces activités qui font notre vie moderne.{' '}
+									</p>
+								</Trans>
+							</div>
+						</details>
+					</li>
+				</ul>
+				<ButtonLink
+					href="/questions-frequentes"
+					className="rounded-full mt-5 mb-10"
```

Comment:
```suggestion
					className="rounded-3xl mt-5 mb-10 border-[1px] !px-4"
```

* le passage au 3xl semble arrondir complètement le bouton, étonnant que le `rounded-full` ne le fasse pas 🤔
* on peut amincir un peu l'épaisseur de la bordure 
* on peut aussi ajouter un peu de padding sur les côtés d'après la maquette